packages/forms/src/templates/ArrayFieldTemplate.js

```diff
@@ -57,23 +57,24 @@ if (process.env.NODE_ENV !== 'production') {
 
 function ArrayFieldTemplate(props) {
 	const { items, canAdd, onAddClick, minItems, maxItems } = props;
+	const addBtnClass = classNames(theme.addBtn, 'btn', 'btn-info');
 	return (
 		<div className={theme.ArrayFieldTemplate}>
 			<IconsProvider />
-			{items &&
-				items.map(element =>
-					<FieldTemplate element={element} cantDelete={items.length <= minItems} />
-				)}
 			{canAdd &&
 				<button
-					className="btn btn-info"
+					className={addBtnClass}
 					type="button"
 					name="btn-new-element"
 					disabled={items.length >= maxItems}
 					onClick={onAddClick}
 				>
 					{`NEW ${props.type}`}
 				</button>}
+			{items &&
+				items.map(element =>
+					<FieldTemplate element={element} cantDelete={items.length <= minItems} />
+				)}
```

Comment:
map need key

Reply:
eslint doesn't require it.

Reply:
but you know we should do it, may be a false positive.